twill-core/src/main/java/org/apache/twill/internal/kafka/client/SimpleKafkaConsumer.java

```diff
@@ -68,7 +69,7 @@
 /**
  * A {@link KafkaConsumer} implementation using the scala kafka api.
  */
-final class SimpleKafkaConsumer implements KafkaConsumer {
+public final class SimpleKafkaConsumer implements KafkaConsumer {
```

Comment:
I wanted to use the `getLastOffset` method in the test. Haven't figured out a way to extract the logic in `getLastOffset` to other places without exposing the `SimpleKafkaConsumer` class because `getLastOffset` uses a lot of private members in `SimpleKafkaConsumer`